jujugui/static/gui/src/app/utils/bakery-utils.js

```diff
@@ -64,12 +63,15 @@ YUI.add('bakery-utils', function(Y) {
     @param {Function} charmstoreCookieSetter A function that can be used to
       send macaroons to the charm store so that they are stored as cookies.
     @param {Object} webHandler The HTTP client that will be used by the bakery.
+    @param {Object} BakeryStorage A reference to the BakeryStorage constructor.
```

Comment:
It's here so that it doesn't have to rely on globals and it'll make it easier to stub for testing.